docker/README.md

```diff
@@ -0,0 +1,23 @@
+# Docker build environment for Jenkins packaging
+
+This offers a simplified build environment for testing packaging
+
+To build the image:
+`docker build -t jenkins-packaging-builder:0.1 .`
```

Comment:
Doesn't really add anything, in this case -- you can't really use a common variable across the readme, etc. 

If we change it, we can bump this.